meson.build

```diff
@@ -8,14 +8,17 @@ project(
 
 os = build_machine.system()
 cc = meson.get_compiler('c')
-lib_m = cc.find_library('m')
-lib_rt = cc.find_library('rt')
 
 alsa_required = false
 if get_option('alsa_in_out').enabled() or get_option('zalsa').enabled()
   alsa_required = true
 endif
 
+lib_rt_required = false
+if get_option('zalsa').enabled()
```

Comment:
PR looks good but I think this check is incorrect.
It passes now, but can give problems in the future.

Say we add some other tool that also makes use of -lrt on linux, but this one is cross-platform.
We cannot require librt non !linux systems, because it wont be available there, but this new tool could still be compiled without it.

So somehow librt needs to be required, except on all systems that do not have it.
How this autodetection works I have no idea.
Maybe always make it optional? Is there even a way for a linux install to have gcc and build tools but not librt?

Reply:
I'm not sure whether this hypothetical is really something we should keep in mind here.
After all: Additions rarely happen. If they happen we can change the build system accordingly.

Currently the only consumer of lib_rt is zalsa. By default, if lib_rt can not be found zalsa will not be built.
However, if the user specifically *requests* that they want to build zalsa and lib_rt is not found, then the configure will fail.

I don't really know under which circumstances librt is available on a system. I guess it is only systems that use glibc?

Reply:
well we can always go with the approach of "let it be until someone complains", which would work here.
maybe we add an Alpine CI test? (for the not-using-glibc usecase)
IIRC it uses `pkg add <name>` for adding new packages

Reply:
well alpine doesnt have the zita libs, so doesnt matter for now.
will just merge, thanks